qiskit/visualization/circuit/text.py

```diff
@@ -1337,15 +1343,16 @@ def add_control_flow(self, node, layers, wire_map):
     def draw_flow_box(self, node, flow_wire_map, section, circ_num=0):
         """Draw the left, middle, or right of a control flow box"""
 
-        conditional = section == CF_LEFT and not isinstance(node.op, ForLoopOp)
+        op = node.op
+        conditional = section == CF_LEFT and not isinstance(op, ForLoopOp)
         depth = str(self._nest_depth)
         if section == CF_LEFT:
-            if isinstance(node.op, IfElseOp):
+            if isinstance(op, IfElseOp):
```

Comment:
It'd be a little better not to make this `op = node.op` substitution, because it's crowding the review diff a bit - it's not so immediately obvious what's relevant to the PR.

Reply:
Done in https://github.com/Qiskit/qiskit/pull/10842/commits/e1d7982f3177d70010e4c1e264d6c9fecd6aedc5.